meta.yaml

```diff
@@ -0,0 +1,98 @@
+{% set setup_py = load_setup_py_data() %}
+package:
+    name: tardis
+    version: {{ setup_py.get('version') }}
+
+source:
+    git_url: https://github.com/tardis-sn/tardis.git
+    git_branch: master
+
+build:
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    host:
+        - setuptools
+        - python=3
```

Comment:
this is bad - we shouldn't manually put this in here.

Reply:
My bad, this is old and unnecessary file. Should be discarded.